lab/lablib/SoundManager.js

```diff
@@ -171,6 +180,51 @@ export class SoundManager {
     );
   }
 
+  /**
+   * Get the current value of a parameter from the currently playing score
+   * @param {string} param_id The ID of the parameter in the score
+   * @returns {number | undefined} The current value of the parameter, or undefined if no value is set
+   */
+  get_param(param_id) {
+    const tween_list = this.score_tweens[this?.current_score]?.[param_id];
+    if (tween_list === undefined || tween_list.length === 0) {
+      return undefined;
+    }
+
+    // TODO: This would be a good case for binary search
```

Comment:
I did a perf check, so far this search isn't noticeable. So just remove this comment and make a note about performance (low-priority)